base/setup/reactos/lang/en-US.rc

```diff
@@ -28,20 +28,20 @@ CAPTION "ReactOS Setup"
 FONT 8, "MS Shell Dlg"
 BEGIN
     AUTORADIOBUTTON "&Install ReactOS", IDC_INSTALL, 7, 20, 277, 10, WS_GROUP | WS_TABSTOP
-    LTEXT "Install a new copy of ReactOS. This option does not keep your files, settings and programs. You can make changes to disks and partitions.", IDC_INSTALLTEXT, 19, 36, 279, 27, NOT WS_GROUP
+    LTEXT "Create a clean installation of ReactOS on your device. This option will not keep your files, settings, and programs. You can make changes to disks and partitions.", IDC_INSTALLTEXT, 19, 36, 279, 27, NOT WS_GROUP
     AUTORADIOBUTTON "&Update or repair ReactOS", IDC_UPDATE, 7, 80, 277, 10
-    LTEXT "Update or repair an installed copy of ReactOS. This option keeps your files, settings and programs. This option is only available if ReactOS is already installed on this computer.", IDC_UPDATETEXT, 19, 96, 279, 27, NOT WS_GROUP
+    LTEXT "Update or repair an installation of ReactOS. This option will keep your files, settings, and programs. This option is only available if ReactOS is already installed on this device.", IDC_UPDATETEXT, 19, 96, 279, 27, NOT WS_GROUP
 END
 
 IDD_UPDATEREPAIRPAGE DIALOGEX 0, 0, 317, 143
 STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU
 CAPTION "ReactOS Setup"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT       "The ReactOS Setup can upgrade one of the available ReactOS installations listed below, or, if a ReactOS installation is damaged, the Setup program can attempt to repair it.", IDC_STATIC, 6, 6, 300, 18
+    LTEXT       "The ReactOS Setup can upgrade one of the available ReactOS installations listed below, or, if an installation is damaged, attempt to repair it.", IDC_STATIC, 6, 6, 300, 18
```

Comment:
```suggestion
    LTEXT       "The ReactOS Setup can upgrade one of the available ReactOS installations listed below, or, if an installation is damaged,  Setup can attempt to repair it.", IDC_STATIC, 6, 6, 300, 18
```

Reply:
Suggesting this because the new language make this sounds like it's what's GOING to happen.

This is extremely optional and isn't specifically advised currently.